docs/rule/require-media-caption.md

```diff
@@ -0,0 +1,41 @@
+# require-media-caption
+
+Captions provide a text version of the spoken and non-spoken audio information for media. They are essential for making audio and video content accessible for users who are deaf as well as those for whom the media is unavailable (similar to `alt` text on an image when it is unable to load).
+
+Captions should contain all relevant information needed to help users understand the media content, which may include a transcription of the dialogue and descriptions of meaningful sound effects. They are synchronized with the media to allow users access to the portion of the content conveyed via the audio track. Note that when audio or video components include the `muted` attribute, however, captions are *not* necessary.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<audio></audio>
+```
+
+```hbs
+<video><track /></video>
+```
+
+```hbs
+<video><track kind="descriptions" /></video>
+```
+
+This rule **allows** the following:
+
+```hbs
+<audio><track kind="captions"></audio>
+```
+
+```hbs
+<video muted="true"></video>
+```
+
+```hbs
+<video muted></video>
+```
+
```

Comment:
I think we can remove this example. We don't really need to show different syntaxes for the same thing.
```suggestion
```